packages/syntax-floating-components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,160 @@
+import { useState, useRef, useEffect } from "react";
+import { StoryObj, Meta } from "@storybook/react";
+import { Tooltip, TooltipContent, TooltipTrigger } from "./Tooltip";
+import styles from "./Tooltip.module.css";
+import type { Placement, Strategy } from "@floating-ui/react";
+import { FloatingDelayGroup, shift } from "@floating-ui/react";
+import image from "../../../../apps/storybook/assets/images/book.svg";
+
+export default {
+  title: "Floating-Components/Tooltip",
+  component: Tooltip,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/%F0%9F%93%90-Syntax?node-id=1007-4109&t=1jAyqXTkYBP57oZL-0",
+    },
+  },
+  argTypes: {
+    children: {
+      control: { type: null },
+    },
+    initialOpen: {
+      control: { type: null },
+    },
+    open: {
+      control: { type: null },
+    },
+    placement: {
+      control: { type: "select" },
+      defaultValue: "right",
+      options: [
+        "top",
+        "top-start",
+        "top-end",
+        "right",
+        "right-start",
+        "right-end",
+        "bottom",
+        "bottom-start",
+        "bottom-end",
+        "left",
+        "left-start",
+        "left-end",
+      ],
+      table: {
+        defaultValue: { summary: "right" },
+      },
+    },
+    strategy: {
+      control: { type: "select" },
+      options: ["absolute", "fixed"],
+      table: {
+        defaultValue: { summary: "absolute" },
+      },
+    },
+    tooltipText: {
+      control: { type: "text" },
+      table: {
+        defaultValue: { summary: "This is a book" },
+      },
+    },
+  },
+  tags: ["autodocs"],
+} as Meta<any>;
+
+const ControlledTooltip = ({ args }: any) => {
+  const [open, setOpen] = useState(false);
+  const imgRef = useRef(null);
+  const scrollableRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (scrollableRef && scrollableRef.current) {
+      scrollableRef.current.scrollTop =
+        scrollableRef.current.scrollHeight / 2 -
+        scrollableRef.current.offsetHeight / 2;
+    }
+  }, [scrollableRef]);
+
+  return (
+    <div
+      ref={scrollableRef}
+      style={{
+        padding: "8px",
+        overflowY: "auto",
+        width: "250px",
+        height: "300px",
+      }}
+    >
+      <div
+        style={{
+          height: "300px",
+          width: "1px",
+        }}
+      />
+      <div style={{ paddingLeft: "100px" }}>
+        <FloatingDelayGroup delay={200}>
+          {/*eslint-disable-next-line @typescript-eslint/no-unsafe-member-access*/}
+          <Tooltip
+            open={open}
+            onOpenChange={setOpen}
+            placement={args.placement as Placement}
```

Comment:
@JFChoe can we get rid the `as Placement`  and the `@typescript-eslint/no-unsafe-member-access` disable